Common/StreamWrapper/tests/Common/TestPayload.cs

```diff
@@ -40,11 +40,11 @@ public enum TaskType
       DatadepCompute,
     }
 
-    public byte [] DataBytes { get; set; }
+    public byte[]? DataBytes { get; set; }
 
     public TaskType Type { get; set; }
 
-    public string ResultKey { get; set; }
+    public string? ResultKey { get; set; }
```

Comment:
Do you really need to distinguish between null and empty? couldn't empty be enough to encode a missing key?